firebase-vscode/src/workflow.ts

```diff
@@ -230,6 +234,43 @@ export async function setupWorkflow(
     broker.send("notifyPreviewChannelResponse", { id: response });
   });
 
+  broker.on(
```

Comment:
do we want all the emulator registration in workflow (what does workflow mean)? I think the amount of code warrants an emulator.ts file.